packages/hui-pages/src/routes/CheckboxDocs/index.js

```diff
@@ -14,13 +14,18 @@ const CheckboxDocs = () => (
       subtitle='A Hixme UI styled checkbox component'
     />
     <Grid fluid>
-      <Row center='xs' style={{ marginBottom: '40px' }}>
+      <Row center='xs' middle='xs'>
         <Col xs>
           <Checkbox defaultChecked text='Checkbox with a label' />
         </Col>
         <Col xs>
           <Checkbox defaultChecked small text='Small checkbox with a label' />
         </Col>
+        <Col xs>
+          <Checkbox defaultChecked jumbo text='Jumbo checkbox' />
```

Comment:
Should add the `<Checkbox jumbo />` example in the `<Code />` block, as well as the `jumbo` prop to the table and README

Reply:
Will do